src/app/App.js

```diff
@@ -1,4 +1,5 @@
 import createWhiteButtonWithIcon from './components/ButtonWhiteWithIcon';
+import VerifyingAnswerIsCorrect from './Components/VerifyingIfAnswerIsCorrect';
```

Comment:
I think this import is not needed unless the module is actually used.

Reply:
Now is without import.